bin/minifollowups/pycbc_sngl_minifollowup

```diff
@@ -124,9 +126,26 @@ insp_data_seglists.coalesce()
 num_events = int(workflow.cp.get_opt_tags('workflow-sngl_minifollowups',
                  'num-sngl-events', ''))
 
+# This helps speed up the processing to ignore a large fraction of triggers
+snr_mask = None
+if args.min_snr:
+    logging.info('Calculating Prefilter')
+    f = hdf.HFile(args.single_detector_file, 'r')
+    idx, _ = f.select(lambda snr: snr > args.min_snr,
+                      '{}/snr'.format(args.instrument),
+                      return_indices=True)
+    snr_mask = numpy.zeros(len(f['{}/snr'.format(args.instrument)]),
+                           dtype=bool)
+    snr_mask[idx] = True
```

Comment:
I think this should go into `SingleDetTriggers` rather than in the executable.

Reply:
In what way would make sense? Just for SNR?

Reply:
It just feels weird to read the file in as one class to get one piece of information and then to open the file again in a different class to get another.

Reply:
I agree. There's already an existing 'filter_func' in SingleDetTriggers, but it doesn't really do the right thing here. That's the 'FIXME' I added in, as if it should be unified in some way. I couldn't figure out a clean way to maintain its syntax and do what's needed.